tutorials/hello/hello_client/src/app/plugins/hello_info/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const DB_KEY_ADDRESS_INFO = Buffer.from('helloInfo:address', 'utf8');
+export const DB_LAST_COUNTER_INFO = Buffer.from('helloInfo:counter', 'utf8');
+export const DB_LAST_HEIGHT_INFO = Buffer.from('helloInfo:height', 'utf8');
```

Comment:
```suggestion
export const DB_KEY_ADDRESS_INFO = Buffer.from([0]);
export const DB_LAST_COUNTER_INFO = Buffer.from([1]);
export const DB_LAST_HEIGHT_INFO = Buffer.from([2]);
```
key doesn't need to be long readable string